lxd/storage/drivers/driver_common.go

```diff
@@ -16,6 +16,7 @@ import (
 
 type common struct {
 	name        string
+	magic       int
```

Comment:
@brauner oh just noticed this, is this used anywhere? I can't see it.

Reply:
Nope, good catch. I also spotted another one.